### PR TITLE
[GR-42167] Add section on build notifications.

### PR DESCRIPTION
**Pull request by graalvmbot**

### CODE REVIEW

**Comment by zakkak** [comment]
Relates to #5327 
Supersedes #5331

File: docs/reference-manual/native-image/README.md [hunk 1]
```diff
@@ -217,12 +217,45 @@ native-image [options] --module <module>[/<mainclass>] [options]
...
+If you are building your project in the background, consider using a command that notifies you when the build process is completed.
+Below, example commands are listed per operating system:
+
+#### Linux
+```bash
+# Ring the terminal bell
+native-image -jar App.jar ... ; printf '\a'
+
+# Use Zenity to open an info dialog box with text
+native-image -jar App.jar ... ; zenity --info --text="GraalVM Native Image build completed with exit code $?"
```
[the remark this answers is not in the document]
Comment (fniephaus) [reply docs/reference-manual/native-image/README.md:232]: Thanks, will add this to one of my next PRs. :)

[the remark this answers is not in the document]
Comment (fniephaus) [reply docs/reference-manual/native-image/README.md:232]: See https://github.com/oracle/graal/pull/5634/commits/086be6cacb88edd0ca4e6d7b64b263d3fbae6b3a.

[the remark this answers is not in the document]
Comment (zakkak) [reply docs/reference-manual/native-image/README.md:232]: Thank you @fniephaus !

**Pull request closed without merging**